src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  getArea: () => number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('your error message');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('your error message');
```

Comment:
[CODE STYLE] - The error message should clearly describe the nature of the error. For instance, 'Triangle inequality violated: the sum of the lengths of any two sides must be greater than the length of the remaining side.'